src/utils/helpers.js

```diff
@@ -58,6 +58,22 @@ export function parseJSON(response) {
     })
   }
 
+  if (response.status === 429) {
+    // Only the body of the response is passed back so to catch a 429
+    // we fill the JSON body with some error details to give information
+    // to the consumer
+    const error = {
+      errors: [{ status: 429 }]
+    }
+    return new Promise(resolve => {
+      resolve({
+        status: response.status,
```

Comment:
@craigtweedy Destructure? 😝

Reply:
How do you mean in this instance? Could you add a code suggestion?

Reply:
> How do you mean in this instance? Could you add a code suggestion?

@craigtweedy I was mostly trolling, but see above example.